engines/pytorch/pytorch-engine/src/test/java/ai/djl/pytorch/integration/ALibUtilsTest.java

```diff
@@ -18,17 +18,21 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-public class LibUtilsTest {
+// Ensure this test run first
+public class ALibUtilsTest {
```

Comment:
Why the change from `LibUtils` to `ALibUtils`?

Reply:
The testng use alphabetical order to run the unittest classes. The engine is only initialized once in the JVM. We want make sure this test executed first in all tests.